src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -1,585 +1,518 @@
 import {
-    ComboBox,
     DefaultButton,
```

Comment:
it is not possible to clear the value in the dropdown menus. IF you delete the content and click away it reverts back to the previous value which is not what I would expect
![cant delete](https://user-images.githubusercontent.com/57726991/184447053-fcf383e1-d83e-49bb-a48f-762d7da73cee.gif)
.

Reply:
Not fixed

Reply:
it is the default behavior, I think it should be as is, what do you think @darsney @azimamansuri ?

Reply:
Gross. Combobox sucks.